lib/rubocop/config.rb

```diff
@@ -440,11 +440,11 @@ def base_dir_for_path_parameters
     end
 
     def target_ruby_version
-      @target_ruby_version ||=
+      @target_ruby_version ||= begin
```

Comment:
This change and the corresponding `end` seem unintented? Maybe we could have a new `UnnecessaryBeginEndBlock` cop :smiley:.

Reply:
mmm, I guess, we could have a new cop that suggests extracting `begin .. end` blocks to separate methods, and thus removing the need for `begin`.

But what I'm suggesting here is different and would apply to every ruby version: remove the `begin .. end` block from assignments if they wrap a single statement.